### ISSUE TITLE
cv browser irregularities

### ISSUE THREAD

**Issue by bradfordcondon** [tripal/tripal#421]
### System information

* Tripal Version: 3

### Issue description
im sorting an issue we had on [HWG](https://github.com/statonlab/hardwoods_site/issues/251) where I had duplicated terms in the tree for GO.

note molecular function regulator:  

![image](https://user-images.githubusercontent.com/7063154/41175152-25ce52e4-6b2a-11e8-82a8-4104c6b4e5df.png)

i went on the demo site to look for this term but didnt see it at all!  http://demo.tripal.info/3.x/cv/lookup/GO

heres the EBI entry:  https://www.ebi.ac.uk/ols/ontologies/go/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0098772

![screen shot 2018-06-08 at 2 44 07 pm](https://user-images.githubusercontent.com/7063154/41175236-6d39ab42-6b2a-11e8-9c84-a8da0794f642.png)

it should actually show up in tWO places: once under molecular function (direct child) and once under biological process ( ->b. regulation -> regulation of mo. function -> mo. function regulator).

So: does hte live site have the full GO loaded, or a subset?

Should the tree browser make sure it doesnt add duplicate terms?  How do the dupe terms get added? Is it a problem with our Chado records and if so what table?  If not, how can browser be corrected?

**Comment by bradfordcondon** [comment]
first things first i guess we need to know if hte demo site has the full GO vocab loaded or not: @spficklin any ideas?

**Comment by bradfordcondon** [comment]
took another look at this: easy fix.

`tripal_chado_vocab_get_term_children` did not select distinct in this query:

```php
 // Get the children.
  $sql = "
    SELECT subject_id
    FROM {cvterm_relationship} CVTR
    WHERE object_id = :object_id
  ";
```

when there were multiple relationship *types* in cvterm_relationship (ie the same subject/object ID's but distint type_ids) they got counted twice.

**Comment by spficklin** [comment]
This looks like it's fixed from PR #492
